src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -816,6 +818,70 @@ def _add_ingress_appgw_addon_role_assignment(result, cmd):
                                    'specified in %s addon. '
                                    'Are you an Owner on this subscription?', vnet_id, CONST_INGRESS_APPGW_ADDON_NAME)
 
+def aks_maintenanceconfiguration_list(cmd, # pylint: disable=too-many-locals,too-many-statements,too-many-branches
```

Comment:
Can you put these new functions into a separate file? Custom.py is already too large and this logic could nicely go into a maintenance.py file (or some similar naming)